Bottom: aca8d9f9f8f392df91992beb3af4c1449f37f67e
Top:    73bceecc37f03c1110df6cf3719bfec3e38dff0a
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-14 07:58:44 +1300

Refresh of stream-summary

---

diff --git a/testtools/tests/test_testresult.py b/testtools/tests/test_testresult.py
index 528e098..5f49bfe 100644
--- a/testtools/tests/test_testresult.py
+++ b/testtools/tests/test_testresult.py
@@ -755,7 +755,10 @@ class TestStreamSummary(TestCase):
         result.stopTestRun()
         self.assertEqual(True, result.wasSuccessful())
         self.assertEqual(3, result.testsRun)
+
+    def test_stopTestRun_inprogress_test_fails(self):
         # Tests inprogress at stopTestRun trigger a failure.
+        result = StreamSummary()
         result.startTestRun()
         result.status("foo", "inprogress")
         result.stopTestRun()
